validator/client/src/test/java/tech/pegasys/teku/validator/client/ValidatorRegistratorTest.java

```diff
@@ -430,6 +432,11 @@ void retrievesCorrectGasLimitForValidators() {
         .thenReturn(Optional.empty());
     when(validatorConfig.getBuilderRegistrationDefaultGasLimit()).thenReturn(defaultGasLimit);
 
+    when(validatorRegistrationPropertiesProvider.getGasLimit(validator2.getPublicKey()))
```

Comment:
We can put these new custom gas limit overrides for validator2 and validator3 where the previous ones are and delete the old ones `when(proposerConfig.getBuilderGasLimitForPubKey....` since they are no longer applicable